gpMgmt/bin/gppylib/logfilter.py

```diff
@@ -273,6 +273,13 @@ def next(self):
         #we need to make a minor format change to the log level field so that
         # our single regex will match both.
         item[16] = item[16] + ": "
+
+        # There is a text containing \n or | symbols. We need to add quotation 
+        # marks, for parsers to process csv text correctly.
+        for i in range(len(item)):
+            if '\n' in item[i] or '|' in item[i]:
```

Comment:
@t1mursadykov pls check that `|` is the only possible csv delimiter for gplogfilter`s output

Reply:
you might use another more readable predicate for verifying whether some character is in string - by [link](https://stackoverflow.com/questions/19463556/string-contains-any-character-in-group#answer-19463605 )

Reply:
Do we really need to use another method to check for a character in the string? I think the current way is readable.

Reply:
I think we might assign `|` symbol to some variable to explicitly name it and to prepare to easy delimiter modification further.